src/login/AzureLoginHelper.ts

```diff
@@ -320,7 +327,7 @@ export class AzureLoginHelper {
 	}
 
```

Comment:
I think we should remove `getStoredCredentials` from this file and put it into `AdalAuthProvider.loginSilent` since MSAL doesn't need it. In fact, you should be able to remove all references to `credentialsSection` and `tryGetKeyTar` in the "common" files (aka anything outside the `msal` and `adal` folders) since each library has their own caching mechanism (MSAL does it through `cachePlugin` and ADAL makes us do everything manually). My comment around `TokenCache.removeAccount` is related to this, btw.